app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_and_belongs_to_many :review_requests
+  has_many :owned_review_requests, class_name: "ReviewRequest", foreign_key: "owner_id", dependent: :destroy, inverse_of: :owner
+
+  def self.create_or_find (json)
```

Comment:
Honestly, I don't think the model should know about a json structure coming from Github. I'd rather leave this code in the service. What do you think?